executor/point_get.go

```diff
@@ -65,6 +65,15 @@ type PointGetExecutor struct {
 	lock     bool
 }
 
+// ReBuildRange used for cached execution for prepared point get plan
+func (e *PointGetExecutor) ReBuildRange(newIdxVals []types.Datum, newHandle int64, newStartTs uint64) error {
```

Comment:
I think `init` is a better name, it should be used in the executor builder.

Reply:
Current implementation is not a totally `Init` meaning, it does only reset the inherited part fields. The new object allocted with `Newxxx` function,  and inited with `Init` function, 
```
func buildPointGet(param) xxx {
    newObj = Newxxx()
    err  = newObj.Init(param)
}

func (a *type） Init(param) error {
      err = base.Init(param)
      init a with param
}
```

Reply:
@coocood  PTAL